plasmapy/mathematics/tests/test_dispersion.py

```diff
@@ -1,92 +1,167 @@
 """Tests for the plasma dispersion function and its derivative"""
 
 import numpy as np
+from numpy import pi as π
 import pytest
 from astropy import units as u
 from ..mathematics import plasma_dispersion_func, plasma_dispersion_func_deriv
 
 
-def test_plasma_dispersion_func():
-    r"""Test the implementation of plasma_dispersion_func against exact
-    results, quantities calculated by Fried & Conte (1961), and
-    analytic results.
-    """
+# (ζ, expected)
+plasma_dispersion_func_table = [
+    (0, 1j * np.sqrt(π)),
+    (1, -1.076_159_013_825_536_8 + 0.652_049_332_173_292_2j),
+    (1j, 0.757_872_156_141_311_87j),
+    (1.2 + 4.4j, -0.054_246_146_372_377_471 + 0.207_960_589_336_958_13j),
+    (9.2j, plasma_dispersion_func(9.2j * u.dimensionless_unscaled)),
+    (5.4 - 3.1j, -0.139_224_873_051_713_11 - 0.082_067_822_640_155_802j),
+    (9.9 - 10j, 2.013_835_257_947_027_6 - 25.901_274_737_989_727j),
+    (4.5 - 10j, -1.367_495_046_340_094_7e35 - 6.853_923_234_842_270_6e34j)]
+
+
+@pytest.mark.parametrize('ζ, expected', plasma_dispersion_func_table)
+def test_plasma_dispersion_func(ζ, expected):
+    r"""Test the implementation of plasma_dispersion_func against
+    exact results, quantities calculated by Fried & Conte (1961),
+    symmetry properties, and analytic results."""
+
+    Z0 = plasma_dispersion_func(ζ)
```

Comment:
I noticed `plasma_dispersion_func()` is defined in terms of `erf()`, perhaps [Dawson function](https://en.wikipedia.org/wiki/Dawson_function) would be cleaner? It's in scipy as [dawsn](https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.dawsn.html).